app/js/results/smaaResultsController.js

```diff
@@ -41,6 +52,43 @@ define(['clipboard', 'require'], function(Clipboard) {
         $scope.state = SmaaResultsService.addSmaaResults($scope.state);
       });
     }
+
+    function initUncertaintyOptions() {
+      $scope.hasNoStochasticMeasurements = SmaaResultsService.hasNoStochasticMeasurements($scope.aggregateState);
+      $scope.hasNoStochasticWeights = SmaaResultsService.hasNoStochasticWeights($scope.aggregateState);
+
+      if (!$scope.scenario.state.uncertaintyOptions) {
```

Comment:
So the options are saved to the scenario? I guess to make them persistent when you `loadState` again. This might cause these settings to leak into the database, or is that intentional?

Reply:
Yes, that is intentional. We suppose the user would like his uncertainty settings saved when switching tabs/reloading.

Reply:
:+1: trying to think ahead a bit to where it might cause issues, seems fine